### PR TITLE
1127 update UI tests

### PR DESCRIPTION
**Pull request by nicolas-chaix-didomi**
Closes: CMP-1127

### CODE REVIEW

**Review by pmerlet-at-didomi — APPROVED** [review]

File: example/integration_test/get_user_consent_test.dart [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    isReady = true;
+  };
+
+  DidomiSdk.addEventListener(listener);
+
+  group("User Consent for Vendor", () {
+    /*
+     * Without initialization
+     */
+
```
Comment (felipe-saez) [inline example/integration_test/get_user_consent_test.dart:51]: this newline doesn't seem to be required.

File: example/integration_test/get_user_consent_test.dart [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+      assertNativeMessage("getUserConsentStatusForVendorAndRequiredPurposes", disabledStatusForVendorAndRequiredPurposesMessage);
+
+      assert(isError == false);
+      assert(isReady == true);
+    });
+
+    /*
+     * With initialization + agree then reset
+     */
+
```
Comment (felipe-saez) [inline example/integration_test/get_user_consent_test.dart:125]: this newline doesn't seem to be required.

File: example/integration_test/get_user_consent_test.dart [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+      assertNativeMessage("getUserConsentStatusForVendorAndRequiredPurposes", disabledStatusForVendorAndRequiredPurposesMessage);
+
+      assert(isError == false);
+      assert(isReady == true);
+    });
+
+    /*
+     * With initialization + disagree then reset
+     */
+
```
Comment (felipe-saez) [inline example/integration_test/get_user_consent_test.dart:194]: this newline doesn't seem to be required.

File: example/integration_test/user_li_test.dart [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+      assertNativeMessage("getUserLegitimateInterestStatusForVendorAndRequiredPurposes", notReadyMessage);
+
+      assert(isError == false);
+      assert(isReady == false);
+    });
+
+    /*
+     * With initialization
+     */
+
```
Comment (felipe-saez) [inline example/integration_test/user_li_test.dart:85]: this newline doesn't seem to be required.

**Review by felipe-saez — APPROVED** [review]

**Review by felipe-saez — APPROVED** [review]

**Pull request merged**